baseband/vdif/header.py

```diff
@@ -194,7 +194,7 @@ def fromvalues(cls, edv=False, **kwargs):
         bits_per_sample : from ``bps``
         frame_length : from ``samples_per_frame`` or ``framesize``
         lg2_nchan : from ``nchan``
-        ref_epoch, seconds, frame_nr : from ``time`` (may need ``sample_rate``)
+        ref_epoch, seconds, frame_nr : from ``time`` and ``sample_rate``
```

Comment:
I think this change is incorrect - `sample_rate` still is not needed if one is at an integer second.

Reply:
My annoyance is that the code doesn't verify that setting time worked, so "may need ``sample rate``" isn't a particularly useful statement, but that's the general problem with setting a header with so many derived properties.

Reply:
Which code does not verify? Don't we get an error if it didn't work?

Reply:
Sorry, didn't fully understand your comment.  Right now `set_time` doesn't throw an error if `self.sample_rate` is `0*u.kHz` for EDV 1, 3.  This can happen when the user only passes a `time`.

Reply:
Ah, yes, of course. I think the assert belongs in `set_time`